keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -228,6 +228,12 @@ N/A
   When `v1` gets introduced:
   - Mark `v1beta2` as deprecated.
 
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?**
```

Comment:
Will take a look tomorrow.

In the meantime - please also adopt to new format of Test Plan:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#test-plan

Reply:
Added the new test plan, PTAL